Add inspo pages to Pay and Benefits subcat (#4209)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/pay-and-benefits/abigails-favourite-things-about-teaching.md

---
title: Abigail's favourite things about teaching
title_paragraph: |-
  <p>Abigail is a maths teacher from Wigan.</p>
description: |-

>   Hear what Abigail enjoys most about teaching, and the impact she can have through her job.

Reviewer: ```suggestion
  Hear what Abigail enjoys most about teaching, and the impact she has through her job.
```

layout: "layouts/minimal" 
colour: grey-pink 

> image: "static/images/content/case-studies/abigail-beeley.jpg" 

Author: ```suggestion
image: "static/images/content/case-studies/abigail.jpg" 
```
